micro-rdk/src/esp32/camera.rs

```diff
@@ -1,22 +1,56 @@
 #![allow(dead_code)]
-use std::time::Duration;
+use std::{
+    collections::HashMap,
+    sync::{Arc, Mutex},
+    time::Duration,
+};
 
-use crate::common::camera::{Camera, CameraError};
-use crate::esp32::esp_idf_svc::sys::camera_config_t;
-use crate::esp32::esp_idf_svc::sys::camera_config_t__bindgen_ty_1;
-use crate::esp32::esp_idf_svc::sys::camera_config_t__bindgen_ty_2;
-use crate::esp32::esp_idf_svc::systime::EspSystemTime;
-use crate::proto::component::camera;
+use crate::esp32::esp_idf_svc::{
+    sys::camera::{
+        camera_config_t, camera_config_t__bindgen_ty_1, camera_config_t__bindgen_ty_2, camera_fb_t,
+        esp_camera_fb_get, esp_camera_fb_return, esp_camera_init,
+    },
+    systime::EspSystemTime,
+};
+use crate::{
```

Comment:
I think you can fold this in to the above `use crate`

Reply:
done